src/main/java/com/yahoo/bullet/rest/query/SseQueryHandler.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright 2018, Yahoo Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.rest.query;
+
+import com.yahoo.bullet.pubsub.Metadata;
+import com.yahoo.bullet.pubsub.PubSubMessage;
+import com.yahoo.bullet.rest.service.QueryService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.servlet.mvc.method.annotation.SseEmitter;
+
+import java.io.IOException;
+
+/**
+ * Query handler that implements results for SSE - multiple results per query.
+ */
+@AllArgsConstructor
+public class SseQueryHandler extends QueryHandler {
```

Comment:
I know Spring uses "Sse" but can we use "SSE"?

Reply:
Done.